src/Request/Body.php

```diff
@@ -2,32 +2,35 @@
 
 namespace Unirest\Request;
 
-use Unirest\Request as Request;
```

Comment:
What is `CURLFile`?

Reply:
CURLFile is placeholder for a local system file that will be uploaded through CURL command